packages/core/core/src/BundlerRunner.js

```diff
@@ -163,8 +162,8 @@ export default class BundlerRunner {
       bundle.assetGraph.merge(subGraph);
 
       bundle.assetGraph.addEdge({
-        from: bundleGroup
-          ? getBundleGroupId(bundleGroup)
+        from: dependency
```

Comment:
Changing the runtimes api (again). This also means that runtime assets will be a peer to the bundle group, not a child, but they'll still resolve fine. It also unifies things with the raw use-case.